### DISCUSSION TITLE
Style Variants

### DISCUSSION THREAD

**Discussion by dharsan19** [vnphanquang/monkeytype-readme#3] [Q&A] [answered]
Hello, @vnphanquang I'm not sure. Is it possible to modify the dynamic Badge's style variant? I've tried manually adjusting it. However, once the actions have been completed, it returns to the Default flat design. Please aid me with this!

**Comment by vnphanquang** [comment, marked as answer]
@dharsan19 have you try setting the action input option `style`?

https://github.com/dharsan19/dharsan19/blob/main/.github/workflows/monkeytype.yaml#L26

```diff
-          style: 'flat' # option from shields.io
+          style: 'for-the-badge' # option from shields.io
```

**Comment by dharsan19** [comment]
What I'm searching for is this. Thank You @vnphanquang
